src/_locales/zh_CN/messages.yml

```diff
@@ -418,6 +418,15 @@ labelNotifyUpdates:
 labelNotifyUpdatesGlobal:
   description: Option to prioritize global notification option over script's setting.
   message: 忽略每个脚本的提示（编辑器中的“设置”页）
+labelUpdatesParallel:
+  description: 当多个脚本的更新地址为同一个域时，会根据设置的并发数进行分批次更新
+  message: 每个域名并发 $1 个进行更新脚本
+labelAutoPopupHeight:
```

Comment:
Please don't change the keys that are not related to this PR.

The `description` doesn't need to be translated. You can run `yarn i18n` to normalize the translations before pushing.

Reply:
I'm sorry, I haven't done international project development yet, I don't understand this aspect, can you do it for me directly?

@gera2ld

Reply:
Did you mean `labelAutoPopupHeight`?
This is another function that I submitted earlier. After consideration, I may need to open another pull request and submit again, so this problem can be ignored

@gera2ld

Reply:
You should have removed labelAutoPopupHeight and labelAutoPopupHeightHint from this pull request. I just did it for you, so please synchronize your local repository.

Reply:
ok thank you, i never understood the real meaning, sorry

@gera2ld